src/templates/index.html

```diff
@@ -50,7 +50,77 @@ <h1 class="font-header better"> BETTER </h1>
+                                            references
+                                            and extract data from them. You
+                                            can also have two pages open, so you can view the contents of the reference
+                                            at the same
+                                            time</p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div>
+                                <div class="card">
+                                    <img src="assets/summary.png" class="icon-welcome-page" />
+                                    <div class="container">
+                                        <h4><b>Summarize your paper</b></h4>
+                                        <p class="text-card-welcome-page">A text wall? Summarize your paper for each key
+                                            component in a few
+                                            lines! You can also select just a piece of text with the mouse</p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div>
+                                <div class="card">
+                                    <img src="assets/searchs.png" class="icon-welcome-page" />
+                                    <div class="container">
+                                        <h4><b>Explore your paper and Cross-References</b></h4>
+                                        <p class="text-card-welcome-page">Read your paper with an extra gear. Hover the
+                                            references
+                                            and extract data from them. You
+                                            can also have two pages open, so you can view the contents of the reference
+                                            at the same
+                                            time</p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div>
+                                <div class="card">
+                                    <img src="assets/highligh.png" class="icon-welcome-page" />
+                                    <div class="container">
+                                        <h4><b>Highligh commonly used, topic-specific phrasings
+                                            </b></h4>
+                                        <p class="text-card-welcome-page">A paper can involve using many typical
+                                            phrases/sentences
+                                            for introduction, background description and experiment result analyses.
+                                            Identifying and
+                                            highlighting these sentences
```

Comment:
This is not true, we only highlight the keywords. Not phrases or sentences. We could hightlight the whole sentence however.
We have to decide about that.